### PR TITLE
Update DoD. Browsers and Devices

### PR DESCRIPTION
**Pull request by giancorderoortiz**
Update definition of done so that it includes browser and device compatibility. 
First draft. 

Closes GH-1032

### CODE REVIEW

File: _pages/contributing/definition-of-done.md [hunk 1]
```diff
@@ -113,6 +117,27 @@ The coverage report can be found in `./coverage/index.html`.
...
+- Firefox 86.0
+- Safari 14.0.1
+- Edge 88.0.705.63
+
+Version numbers subject to change periodically. Internet Explorer not officially supported. 
+
+## Device Compatibility
+
+New feature must be compatible with major mobile devices in the marketplace. Manual happy path test must pass with: 
+- iPhone 7, 10, X ,11 (at least one)
```
Comment (hackergil) [inline _pages/contributing/definition-of-done.md:133]: I think it's iphone 8. 7 is too old. Also, we should focus on mobile Safari versions.

**Review by hackergil — CHANGES_REQUESTED** [review]
@gladius-mtl to review wording etc

File: _pages/contributing/definition-of-done.md (hunk 1 above)
Comment (giancorderoortiz) [reply _pages/contributing/definition-of-done.md:133]: ok. Will change to iPhone8. 
Safari mobile versions are tied to the iOS version itself afaik.

**Review by gladius-mtl — APPROVED** [review]

File: _pages/contributing/definition-of-done.md (hunk 1 above)
Comment (gladius-mtl) [reply _pages/contributing/definition-of-done.md]: Added that new features "must be compatible with Safari on iOS, and Chrome on Android", and also mentioned that phones and tablets "should be running on the latest versions of their respective operating systems."

**Pull request merged**